### PR TITLE
Rename root element

### PR DESCRIPTION
**Pull request by nllong**

### CODE REVIEW

**Comment by edward-ellis** [comment]
Since I would like to see a schema semantic version number added I agree with @nllong that option B is my preferred option.  The proposal seems to be:
1. Change Facility to Building
2. Change Audit to Facility
3. Optionally, add BuildingSync at the root

What is not clear is if we are excising Audit and leaving audit specific attributes under Facility or moving audit specific attributes somewhere else.  I can't enumerate which attributes are audit specific off the top of my head, but I'm sure there must be *some*.

**Comment by nllong** [comment]
@edward-ellis. Based on feedback I think item 3 in your list is no longer optional, that is we are planning on adding BuildingSync to the root.

Regarding your unclear comment: proposal #58 would be where we put other audit specific data. At the moment it is planned to put it under the BuildingSync element. If you have any feedback on that one, then please comment on that proposal.

File: proposals/2019/Rename Root Element.md [hunk 1]
```diff
@@ -15,38 +15,48 @@ Standard 211-type audit. For example:
...
+* BuildingSync/ (New)
+	* Campuses/Campus/ (Previously `Audit`)
+	    * Sites/Site/ (Unchanged)
+			* Buildings/Building (Previously `Facility`)
+
+*Note: Multiple options for the name of these elements (Campuses/Campus) have been proposed with the intent of finding the most appropriate term while remaining BEDES compliant. The options are:*
+
+1. Campuses/Campus/
+2. Premises/PremisesIdentifer/
+3. Facilities/Facility/
```
[the remark this answers is not in the document]
Comment (aqwolfe) [reply proposals/2019/Rename Root Element.md]: @nllong ok we will change Campuses/Campus to Facilities/Facility

File: BuildingSync.xsd [hunk 2]
```diff
@@ -1,150 +1,438 @@
...
 <xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema" xmlns:auc="http://nrel.gov/schemas/bedes-auc/2014" targetNamespace="http://nrel.gov/schemas/bedes-auc/2014" elementFormDefault="qualified" attributeFormDefault="unqualified">
   <xs:annotation>
     <xs:documentation>BuildingSync Schema - Version 0.3 (May 31, 2018)</xs:documentation>
   </xs:annotation>
-  <xs:element name="Audits">
+  <xs:element name="BuildingSync">
     <xs:complexType>
       <xs:sequence>
-        <xs:element name="Audit" maxOccurs="unbounded">
+        <xs:element name="Campuses">
```
[the remark this answers is not in the document]
Comment (nllong) [reply BuildingSync.xsd]: @aqwolfe -- Can you update this to be facilities/facility? I think that is the right term. I think it resonates more with a facilities manager.

[the remark this answers is not in the document]
Comment (aqwolfe) [reply BuildingSync.xsd]: @nllong yes in process now

**Comment by aqwolfe** [comment]
@nllong should I wait for this to be merged to develop before I start on #58?

**Comment by nllong** [comment]
We need to update the /Buildings/Building/FacilityClassification element to be BuildingClassification. I will do this and update the annotations to remove Facility.

**Pull request merged**